org.wso2.carbon.identity.organization.mgt.core/src/main/java/org/wso2/carbon/identity/organization/mgt/core/OrganizationManagerImpl.java

```diff
@@ -340,11 +340,11 @@ public void patchOrganization(String organizationId, List<Operation> operations)
             throw handleClientException(PATCH_REQUEST_INVALID_ORGANIZATION, null);
         }
         validateOrganizationPatchOperations(operations, organizationId);
-        for (Operation operation : operations) {
-            cacheBackedOrganizationMgtDAO.patchOrganization(organizationId, operation);
-            // Fire post-event
-            fireEvent(POST_PATCH_ORGANIZATION, organizationId, operation, Status.SUCCESS);
-        }
+        cacheBackedOrganizationMgtDAO.patchOrganization(organizationId, operations);
+
+        // Fire post-event (bulk)
+        fireEvent(POST_PATCH_ORGANIZATION, organizationId, operations, Status.SUCCESS);
```

Comment:
There are event handlers written to POST_PATCH_ORGANIZATION. Therefore, should I revert it back  so that the fireEvent will operate on single operations?